syntax="proto3";

service Greeter{
    //单次请求单次回应
    rpc GetMsg(ReqMsg) returns (grpc_dpdk_arp_res){}
}

//request
message ReqMsg{
    string data = 1;
}

//response
message table_pkts {
    string scrip = 1;
    uint32 request_pkts_amount = 2;
    uint32 rate = 3;
}
    
message grpc_dpdk_arp_res{
    repeated table_pkts arp_pkts = 1; 
    repeated table_pkts multicast_pkts = 2;
}
